### PR TITLE
#165: Proposal for PR

### PR DESCRIPTION
**Pull request by Schedulaar**

### CODE REVIEW

File: administration/src/components/generation/EakGeneration.tsx [hunk 1]
```diff
@@ -1,71 +1,94 @@
...
-                        <div className="form-spacing">
-                            <AddEakForm onClick={addForm} />
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <Container>
+            <ButtonBar stickyTop={0}>
+                <Button icon="export" text="QR-Codes generieren" intent="success"/>
```
[the remark this answers is not in the document]
Comment (maxammann) [reply administration/src/components/generation/EakGeneration.tsx:81]: :P

File: administration/src/components/generation/EakGeneration.tsx [hunk 2]
```diff
@@ -1,71 +1,94 @@
...
 
-    /*componentDidMount(): void {
-        const firstFormColumn = document.getElementsByClassName("form-column")[0];
-        const addEakBpCard = document.getElementById("add-eak-form-card")
-        if (addEakBpCard != null) {
-            addEakBpCard.style.height = firstFormColumn.clientHeight - 50 + "px"
-        }
-    }*/
-
     const addForm = useCallback(() => {
```
Comment (maxammann) [inline administration/src/components/generation/EakGeneration.tsx:70]: Actually useCallback is not needed here

Comment (maxammann) [reply administration/src/components/generation/EakGeneration.tsx:70]: My fault for adding it :D

**Review by maxammann — APPROVED** [review]

File: .idea/vcs.xml [hunk 3]
```diff
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="GithubSharedProjectSettings">
-    <option name="branchProtectionPatterns">
-      <list>
-        <option value="main" />
-      </list>
-    </option>
-  </component>
```
Comment (Schedulaar) [inline .idea/vcs.xml:9]: :sob:

**Pull request merged**